### PR TITLE
Use the home domain instead of explicit transfer and auth servers

### PR DESCRIPTION
**Pull request by msfeldstein**
Instead of inputting the bridge server and auth server explicitly, you can now put in just the home domain, and we will fetch the stellar.toml to pull those values.  This will be easier to configure as well as help validate our toml files for this particular use case.
#51

### CODE REVIEW

File: src/steps/check_toml.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    let HOME_DOMAIN = Config.get("HOME_DOMAIN");
+    if (HOME_DOMAIN.indexOf("http") != 0) {
+      HOME_DOMAIN = "https://" + HOME_DOMAIN;
+    }
+    request(`${HOME_DOMAIN}/.well-known/stellar.toml`);
+    const resp = await fetch(`${HOME_DOMAIN}/.well-known/stellar.toml`);
+    const text = await resp.text();
+    const information = toml.parse(text);
+    response(`${HOME_DOMAIN}/.well-known/stellar.toml`, information);
+    expect(information.AUTH_SERVER, "Toml file doesn't contain an AUTH_SERVER");
```
Comment (tomerweller) [inline src/steps/check_toml.js]: I think you mean `WEB_AUTH_ENDPOINT`?
`AUTH_SERVER` is the legacy compliance protocol (SEP-3)

Comment (tomerweller) [reply src/steps/check_toml.js]: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0001.md#account-information

Comment (msfeldstein) [reply src/steps/check_toml.js]: Indeed you're right.  Saldo just had them point to the same place.

Comment (msfeldstein) [reply src/steps/check_toml.js]: fixed

**Pull request merged**